btcec/schnorr/musig2/nonces.go

```diff
@@ -255,17 +255,19 @@ func genNonceAuxBytes(rand []byte, i int,
 		return nil, err
 	}
 
-	switch len(opts.msg) {
+	switch {
 	// If the message isn't present, then we'll just write out a single
 	// uint8 of a zero byte: m_prefixed = bytes(1, 0).
-	case 0:
+	case opts.msg == nil:
```

Comment:
nit: squash this commit with previous one that updates the message encoding logic?